engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/gameDetailsScreen/GameDetailsScreen.java

```diff
@@ -285,7 +285,12 @@ private void setUpGameWorlds() {
         gameWorlds.setItemRenderer(new AbstractItemRenderer<WorldInfo>() {
             @Override
             public void draw(WorldInfo value, Canvas canvas) {
-                canvas.drawText(value.getCustomTitle());
+                if(value.getCustomTitle().isEmpty()) {
```

Comment:
Trivial code style: space missing - should be `if (` :-) Checkstyle should catch (same later around a `=`)